truffle/com.oracle.truffle.api.test/src/com/oracle/truffle/api/vm/ImplicitExplicitExportTest.java

```diff
@@ -270,6 +273,27 @@ public ExportImportLanguage1() {
         }
 
         @Override
+        protected Ctx createContext(Env env) {
+            // BEGIN: arguments.read
+            Object args = env.importSymbol("args");
+            List<?> argsList = null;
+            if (args instanceof TruffleObject) {
+                // convert arguments to list
+                argsList = JavaInterop.asJavaObject(
+                    List.class, (TruffleObject) args
+                );
```

Comment:
Why does the String[] magically becomes a List?
Also, it seems not so practical to use, looking at the lines below (get, cast, etc).